ingress-lb/kontena.yml

```diff
@@ -43,22 +41,25 @@ services:
     # {% endif %}
     deploy:
       strategy: daemon
-      wait_for_port: 80
+      wait_for_port: 1000
       interval: 7d # re-deploy once in a week for security updates
     health_check:
-      protocol: http
-      port: 80
-      uri: /__health
+      protocol: tcp
+      interval: 60
+      port: 1000
```

Comment:
Seems to work fine

```
D, [2017-12-13T09:04:18.709262 #1] DEBUG -- Kontena::Workers::ContainerHealthCheckWorker: checking health for container: /ingress-lb.lb-1 using tcp ip and port: 10.81.128.21:1000
D, [2017-12-13T09:04:18.709711 #1] DEBUG -- Kontena::Workers::ContainerHealthCheckWorker: got status: true
```

Reply:
Although TBH the http healthchecks on the [haproxy `monitor-uri`](https://cbonte.github.io/haproxy-dconv/1.8/configuration.html#4-monitor-uri) might still be a better idea than just a TCP healthcheck on port 1000 - we might even want to make the stats thing on port 1000 optional? It's not that great if you're using `network_mode: host`.

With https://github.com/kontena/kontena-loadbalancer/pull/41 the HTTP healthchecks should no longer fail when installing this stack https://github.com/kontena/kontena/issues/3121